internal/ceb/config.go

```diff
@@ -193,6 +195,58 @@ func (ceb *CEB) watchConfig(
 	}
 }
 
+var sigMap = map[string]os.Signal{
+	"SIGABRT":   unix.SIGABRT,
+	"SIGALRM":   unix.SIGALRM,
+	"SIGBUS":    unix.SIGBUS,
+	"SIGCHLD":   unix.SIGCHLD,
+	"SIGCONT":   unix.SIGCONT,
+	"SIGHUP":    unix.SIGHUP,
+	"SIGINT":    unix.SIGINT,
+	"SIGIO":     unix.SIGIO,
+	"SIGKILL":   unix.SIGKILL,
+	"SIGPIPE":   unix.SIGPIPE,
+	"SIGPROF":   unix.SIGPROF,
+	"SIGQUIT":   unix.SIGQUIT,
+	"SIGSEGV":   unix.SIGSEGV,
+	"SIGSTOP":   unix.SIGSTOP,
+	"SIGSYS":    unix.SIGSYS,
+	"SIGTERM":   unix.SIGTERM,
+	"SIGTRAP":   unix.SIGTRAP,
+	"SIGTSTP":   unix.SIGTSTP,
+	"SIGTTIN":   unix.SIGTTIN,
+	"SIGTTOU":   unix.SIGTTOU,
+	"SIGUSR1":   unix.SIGUSR1,
+	"SIGUSR2":   unix.SIGUSR2,
+	"SIGVTALRM": unix.SIGVTALRM,
+	"SIGWINCH":  unix.SIGWINCH,
+
+	"ABRT":   unix.SIGABRT,
+	"ALRM":   unix.SIGALRM,
+	"BUS":    unix.SIGBUS,
+	"CHLD":   unix.SIGCHLD,
+	"CONT":   unix.SIGCONT,
+	"HUP":    unix.SIGHUP,
+	"INT":    unix.SIGINT,
+	"IO":     unix.SIGIO,
+	"KILL":   unix.SIGKILL,
+	"PIPE":   unix.SIGPIPE,
+	"PROF":   unix.SIGPROF,
+	"QUIT":   unix.SIGQUIT,
+	"SEGV":   unix.SIGSEGV,
+	"STOP":   unix.SIGSTOP,
+	"SYS ":   unix.SIGSYS,
```

Comment:
I saw this was fixed on the other branch but maybe we can fix this here too since it's ready for merge
```suggestion
	"SYS":   unix.SIGSYS,
```